src/crypto.rs

```diff
@@ -384,21 +385,22 @@ mod test {
         };
 
         // Encrypt
-        let (encrypted, key) = encrypt_file_data(&data);
+        let (encrypted, key) = encrypt_file_data(&data).unwrap();
         let encrypted_thumb = encrypted.thumbnail.expect("Thumbnail missing");
 
+        let crypto_box: XSalsa20Poly1305 = XSalsa20Poly1305::new(&key);
+
         // Ensure that encrypted data is different from plaintext data
         assert_ne!(encrypted.file, file_data);
         assert_ne!(encrypted_thumb, thumb_data);
-        assert_eq!(encrypted.file.len(), file_data.len() + secretbox::MACBYTES);
```

Comment:
I could not find a MAC size variable anywhere in the crates that I used and therefore removed those assertions